docs/using-radicle/contributing.md

```diff
@@ -0,0 +1,47 @@
+---
+id: contributing
+title: Contributing to projects
+---
+
+To contribute to another user's project, you must first find it on the Radicle
+network. To find a project, you need its [**Radicle ID**][ri]. This can be found
+in the project page under the [project name][pn], or in a seed node dashboard
+such as [seedling.radicle.xyz](seedling.radicle.xyz).
+
+![Identity][id]
+
+Once you have the Radicle ID, copy it into the Upstream search bar. From there,
+following the project will replicate it to your local machine once it's found in
+your network of connected peers. You won't see the project metadata — such as,
+the name of the project — until it is verified by Radicle Link. 
+
+![Search bar][sb]
+![Found Project][fp]
+
+### Cloning projects
+
+To clone a working copy of a followed project to your computer, you can use the
+**Fork** option within the Upstream client. This will clone the project to a
+specified location on your local machine and publish your version to the Radicle
+network.
+
+![Fork Project][fo]
```

Comment:
Shows the checkout button still, it's now a `Fork` button. Bless @cloudhead

Reply:
fixed it 👍